internal/services/ch/queries.go

```diff
@@ -343,3 +343,15 @@ func getFilterMods(filter *model.SignalFilter) []qm.QueryMod {
 	}
 	return mods
 }
+
+// getDeviceActivityQuery returns the most recent 3 hour bucket during which device activity was recorded.
+func getDeviceActivityQuery(bucketInterval int64, vehicleTokenID int, adManuf string) (string, []any) {
```

Comment:
Do we actually need to add anything to `ch`?

Reply:
What if we use `GetLatestSignals`?

Reply:
<img width="1150" alt="image" src="https://github.com/user-attachments/assets/9503cb87-b74d-4de0-8e12-0fe761b2e5db">